linera-service/tests/end_to_end_tests.rs

```diff
@@ -1127,3 +1147,78 @@ async fn test_end_to_end_social_user_pub_sub() {
     node_service1.assert_is_running();
     node_service2.assert_is_running();
 }
+
+#[test_log::test(tokio::test)]
+async fn test_end_to_end_fungible() {
+    use fungible::{AccountOwner, FungibleInitial};
+    use linera_base::{crypto::CryptoHash, data_types::Amount};
+    use std::collections::BTreeMap;
+    let _guard = INTEGRATION_TEST_GUARD.lock().await;
+
+    let network = Network::Grpc;
+    let mut runner = TestRunner::new(network, 4);
+    let client = runner.make_client(network);
+
+    runner.generate_initial_validator_config().await;
+    client.create_genesis_config().await;
+    let chain = client.get_wallet().default_chain().unwrap();
+    runner.run_local_net().await;
+    let (contract, service) = runner.build_application("fungible2").await;
+
+    // The players
+    let owner_a = client.get_owner().expect("Failed to get the owner");
+    let account_owner_a = AccountOwner::User(owner_a);
+    let owner_b = Owner(CryptoHash::from([23, 22, 145, 21]));
+    let account_owner_b = AccountOwner::User(owner_b);
+    // The initial accounts
+    let mut accounts = BTreeMap::new();
+    let amount_a = Amount::from(5);
+    let amount_b = Amount::from(2);
+    accounts.insert(account_owner_a, amount_a);
+    accounts.insert(account_owner_b, amount_b);
+    let fungible_initial = FungibleInitial { accounts };
+
+    // Setting up the application and verifying
+    let application_id = client
+        .publish_and_create(contract, service, fungible_initial, None)
+        .await;
+    let node_service = client.run_node_service(chain, 8080).await;
+    let chain_id = node_service.chain_id.unwrap();
+
+    let application = node_service.make_application(&application_id).await;
+
+    let value = application
+        .get_fungible_accountowner_amount(&account_owner_a)
+        .await;
+    assert_eq!(value, amount_a);
+
+    // Transfering
+    // Following does not work:
+    //     let account_b = fungible::Account { chain_id, owner: account_owner_b };
+    //     let destination_b = fungible::Destination::Account(account_b);
+    //     let destination_b_str = destination_b.to_value();
+    // For two reasons:
+    // --- The "{{ Account: ..... }}" is put around.
+    // --- The chainId is written chain_id
+    let destination_b_str = format!(
+        "{{chainId: \"{}\", owner: {}}}",
+        chain_id,
+        account_owner_b.to_value()
+    );
+
+    let amount_transfer = Amount::from(1);
+    let query_string = format!(
+        "mutation {{  transfer(owner: {}, amount: {}, targetAccount: {})}}",
+        account_owner_a.to_value(),
+        amount_transfer,
+        destination_b_str
+    );
+    let _reply = application.query_application(&query_string).await;
```

Comment:
It's better to remove `let _reply = ` because the linter will complain if later we change the signature to return an error (or anything that must be consumed).

Reply:
Ok.